lib/repl.js

```diff
@@ -571,7 +567,8 @@ REPLServer.prototype.complete = function(line, callback) {
             this.context.constructor &&
             this.context.constructor.name === 'Context') {
           var contextProto = this.context;
-          while (contextProto = Object.getPrototypeOf(contextProto)) {
+          while (
+            (contextProto = Object.getPrototypeOf(contextProto)) === true) {
```

Comment:
Hmm, while assignment inside a condition is a common error, I like the conciseness of how it was before.